robot/install.yaml

```diff
@@ -8,3 +8,6 @@
   name: networking
 - type: target
   name: vim
+# Disk space analyzer:
+- type: system
+  name: ncdu
```

Comment:
Why this on the robot?

Reply:
When the disk is full, it's near impossible to debug why. Because even cd-ing into directories is hard because tab-complete uses disk-space.
Having this tool pre-installed can be a ~life~robot-saver.